src/ucp/api/ucp.h

```diff
@@ -660,16 +664,30 @@ typedef struct ucp_ep_params {
     uint64_t                field_mask;
 
     /**
-     * Destination address; the address must be obtained using
-     * @ref ucp_worker_get_address.
-     * This field is mandatory for filling (along with its corresponding bit
+     * Destination address; if a connection manager isn't used,
+     * this address must be obtained using @ref ucp_worker_get_address and
+     * this field is mandatory for filling (along with its corresponding bit
      * in the field_mask - UCP_EP_PARAM_FIELD_REMOTE_ADDRESS).
-     * The ucp_ep_create routine will return with an error if the address isn't
+     * The ucp_ep_create routine will return with an error if the
+     * UCP_EP_PARAM_FIELD_ADDRESS_TYPE_CM flag isn't set and this address isn't
      * specified.
      */
     const ucp_address_t     *address;
 
     /**
+     * Destination address in the form of a sockaddr; if a connection manager
+     * is used, this address must be filled and obtained from the user.
+     * The following flags need to be set:
+     * The UCP_EP_PARAM_FIELD_REMOTE_ADDRESS bit in the field_mask
+     * and the UCP_EP_PARAM_FIELD_ADDRESS_TYPE_CM bit in the field_mask indicating
+     * that the type of the remote address is a sockaddr.
+     * The ucp_ep_create routine will return with an error if the
+     * UCP_EP_PARAM_FIELD_ADDRESS_TYPE_CM flag is set but the sock_addr
+     * isn't specified.
+     */
+    const struct sockaddr   *sock_addr;
```

Comment:
We already have one destination address. Can they be used simultaneously? if yes - that's effect? if no - may be reuse existing one just adding address type/format and redefine it as 'void*'?